### DISCUSSION TITLE
add GLwrapper

### DISCUSSION THREAD

**Discussion by fheroes2bugs** [ihhub/fheroes2#7946] [Ideas]
Sure, there must be some discussion about GLwrapper:
https://www.old-games.ru/forum/threads/serija-igr-heroes-of-might-and-magic-gl-wrapper-patches.80503/

I just compare, how smooth GLwrapper is:
![glwrapper](https://github.com/ihhub/fheroes2/assets/145407420/0e7feda7-c201-4a91-9a09-fa319befdb44)

Here is the 1080 video with it:
https://www.youtube.com/watch?v=L-VGsgc6LiY

Can it be added as function? For example, press hotkey F7 and GLWrapper is enabled...

**Comment by ihhub** [comment]
A short answer: no. This is a separate application for Windows that adds image scaling and some fancy questionable filtration. We have a topic about adding some filtrations and fheroes2 scales images natively.
